Resources/README.md

```diff
@@ -2,3 +2,91 @@
 
 This folder contains uploads, links, and recommendations from community members. 
 Whether you want to get guidance, explore further, dig deeper, or find a foothold, this is a great place to look. 
+
+## Contents
+* [About this project](/about-this-project)
+* [Friends, supporters, and partners](/friends-supporters-and-partners)
+* [Created by community members](/created-by-community-members)
+* [Autism](/autism)
+  * academic articles
+  * recommended books
+  * blogs and websites
+  * other usefel resources
+* [Open research](/open-research)
+* [Citizen science](/citizen-science)
+* [Technical guides](/technical-guides)
+  * GitHub
+  * UX design
+  * Python
+  * Django
+
+## About this project (videos and presentations)
+
+## Friends, supporters, and partners
+
+* [The Alan Turing Institute](https://www.turing.ac.uk/): The Alan Turing Institute is funded by Autistica to deliver the Autistica/Turing citizen science project and the platform AutSPACEs. 
+It is the UK's national institute for data science and AI innovation. 
+* [Autistica](https://www.autistica.org.uk/): Autistica is a UK autism research charity who support autistic people by funding research, shaping policy and working with autistic people to understand their needs. Autistica upports and funds the Autistica/Turing citizen science project.
```

Comment:
Typo - Autistica **supports** and funds the Autistica/Turing citizen science project.